hedera-node/hedera-mono-service/build.gradle.kts

```diff
@@ -101,6 +101,10 @@ val jmhDaggerSources = file("build/generated/sources/annotationProcessor/java/jm
 
 java.sourceSets["jmh"].java.srcDir(jmhDaggerSources)
 
+val generatedSources = file("build/generated/sources/annotationProcessor/java/main")
+
+java.sourceSets["main"].java.srcDir(generatedSources)
+
```

Comment:
I don't get why this is in this PR.  What's going on here?

Reply:
Avoids some annoying compile errors when the Gradle build doesn't grasp that sources have changed.